library/Fission/Web/User/UpdateExchangeKeys.hs

```diff
@@ -0,0 +1,67 @@
+module Fission.Web.User.UpdateExchangeKeys
+  ( API
+  , AddAPI
+  , RemoveAPI
+  , server
+  , addKey
+  , removeKey
+  ) where
+
+import           Servant
+
+import           Fission.Prelude
+import           Fission.Models
+import           Fission.Authorization
+ 
+import qualified Fission.Web.Error as Web.Error
+import qualified Fission.User      as User
+
+import qualified Crypto.PubKey.RSA as RSA
+
+type API = AddAPI :<|> RemoveAPI
+
+type AddAPI
+  =  Summary "Add Public Exchange Key"
+  :> Description "Add a key to the currently authenticated user's root list of public exchange keys"
+  :> Capture "public-did-key" RSA.PublicKey
+  :> Put     '[JSON] [RSA.PublicKey]
+
+type RemoveAPI
+  =  Summary "Remove Public Exchange Key"
+  :> Description "Remove a key from the currently authenticated user's root list of public exchange keys"
+  :> Capture "public-did-key" RSA.PublicKey
```

Comment:
“Private DID Key” isn’t a thing, so maybe just `did-key` or `did`? Actually, it may not be a bad idea to capture these as DIDs in case we later add more algorithms — can’t hurt since they’re self-describing. Thoughts?

Reply:
yup I like just `did` :+1:

Reply:
I changed the name to `did-key-rsa` since that's the only thing we can accept at the moment. We can update that as we can accept more keys :ok_hand: